modules/virt-runbook-vmcannotbeevicted.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-runbooks.adoc
+
+:_content-type: REFERENCE
+[id="virt-runbook-vmcannotbeevicted_{context}"]
+= VMCannotBeEvicted
+
+// Edited by apinnick, Nov 2022
+
+[discrete]
+[id="meaning-vmcannotbeevicted_{context}"]
+== Meaning
+
+This alert fires when the eviction strategy of a virtual machine (VM) is set
+to `LiveMigration` but the VM is not migratable.
+
+[discrete]
+[id="impact-vmcannotbeevicted_{context}"]
+== Impact
+
+Non-migratable VMs prevent node eviction. This condition affects operations
+such as node drain and updates.
+
+[discrete]
+[id="diagnosis-vmcannotbeevicted_{context}"]
+== Diagnosis
+
+. Check the VMI configuration to determine whether the value of `evictionStrategy`
+is `LiveMigrate` of the VMI:
```

Comment:
Should this be "for the VMI"?

Reply:
Oh boy, that is a really bad sentence. Will rewrite it.